src/components/onboarding/OnboardingPage.tsx

```diff
@@ -0,0 +1,58 @@
+'use client'
+
+import Image from 'next/image'
+import { useRouter } from 'next/navigation'
+
+export default function OnboardingPage() {
+  const router = useRouter()
+
+  const handleGithubConnect = async () => {
+    try {
+      // GitHub OAuth 연동 로직 구현
+      console.log('GitHub 계정 연동 시작')
+
+      // 서버에 GitHub 연동 요청
+      // 연동 성공 시 다음 페이지로 이동
+
+      // 임시로 바로 대시보드로 이동
+      router.push('/dashboard')
+    } catch (error) {
+      console.error('GitHub 연동 오류:', error)
+      // 에러 처리 로직 추가 가능
+    }
+  }
+
+  return (
+    <div className="flex h-full w-full flex-col items-start bg-white px-[60px] pb-[55px]">
+      {/* 그라데이션 배경 */}
+      <div className="absolute bottom-0 left-0 h-[226px] w-full bg-gradient-to-b from-transparent to-slate-100" />
+
+      {/* 메인 컨텐츠 */}
+      <div className="flex h-[689px] w-full max-w-[1280px] flex-col items-center justify-center gap-[35px]">
+        <div className="flex flex-col items-center gap-[17px] py-20">
+          <h1 className="w-full max-w-[928px] text-center text-[52px] leading-[62.05px] font-semibold tracking-widest text-slate-800">
+            GitHub 커밋 기반 맞춤 CS 면접 준비,
+          </h1>
+
+          <h2 className="text-center text-[52px] leading-[62.05px] font-semibold tracking-widest text-slate-800">
+            CoMentor와 함께!
+          </h2>
+        </div>
+
+        {/* 깃허브 연동 버튼 */}
+        <button
+          onClick={handleGithubConnect}
+          className="flex h-[73px] w-[420px] cursor-pointer items-center justify-center gap-[15px] rounded-[33px] bg-slate-800 px-[31px] py-[20px] shadow-[0px_4px_4px_0px_rgba(0,0,0,0.25)] transition-colors hover:bg-slate-700"
+        >
+          <div className="relative h-6 w-6">
+            <Image src="/images/Github.svg" alt="Github Icon" fill priority />
+          </div>
```

Comment:
왜 굳이 div 태그에 relative를 추가해서 Image 크기를 설정하는 지 모르겠음
크기 설정은 Image 태그에서도 width,height 속성을 직접 지정할 수 있는데 이유가 몬지 궁금금

```suggestion
          <Image src="/images/Github.svg" alt="Github Icon" width={24} height={24} />
```

Reply:
그게 원래 나도 그렇게 했었는데
그렇게 작성하고 실행하니까 
Image with src "http://localhost:3000/images/Github.svg" has either width or height modified, but not the other. If you use CSS to change the size of your image, also include the styles 'width: "auto"' or 'height: "auto"' to maintain the aspect ratio.
이런식으로 종횡비가 불일치한? 경고가 발생해서 해결 방법중에

1번
          <div className="relative h-6 w-6">
            <Image src="/images/Github.svg" alt="Github Icon" fill priority />
          </div>
   
2번
          <div className="relative h-6 w-6">
            <Image
              src="/images/Github.svg"
              alt="Github Icon"
              width={24}
              height={24}
              style={{ width: 'auto', height: 'auto' }}
            />
          </div>
          
3번
         <Image
            src="/images/Github.svg"
            alt="Github Icon"
            width={24}
            height={24}
            className="h-6 w-6"
          />
          
 이렇게 옵션이 있었는데, 1번이 제일 유지보수성 측면이랑 next.js가 priority가 있는 이미지를 자동으로 preload해서 최적화에 좋아보여서 1번으로 했었음